eox_nelp/pearson_vue/tests/test_pipeline.py

```diff
@@ -19,11 +19,134 @@
+    def test_check_not_complete_not_graded(self, get_completed_and_graded_mock):
+        """Test the pipeline return expected dict(safely_pipeline_termination) with empty `is_passing`, and
+           is_complete=False, is_graded=False.
+        Expected behavior:
+            - Returned value is dict with `safely_pipeline_termination`
+        """
+        pipeline_kwargs = {}
+        get_complete_and_graded_output = {
+            "is_complete": False,
+            "is_graded": False,
+        }
+        expected_output = {
+            "safely_pipeline_termination": True,
+        }
+        get_completed_and_graded_mock.return_value = (
+            get_complete_and_graded_output["is_complete"],
+            get_complete_and_graded_output["is_graded"],
+        )
+
+        result = terminate_not_full_completion_cases(self.user_id, self.course_id, **pipeline_kwargs)
+
+        self.assertDictEqual(expected_output, result)
+
+    @patch("eox_nelp.pearson_vue.pipeline.get_completed_and_graded")
+    def test_check_not_complete_graded(self, get_completed_and_graded_mock):
+        """Test the pipeline return expected dict(safely_pipeline_termination) with empty `is_passing`, and
+           is_complete=False, is_graded=True.
+        Expected behavior:
+            - Returned value is dict with `safely_pipeline_termination`
+        """
+        pipeline_kwargs = {}
+        get_complete_and_graded_output = {
+            "is_complete": False,
+            "is_graded": True,
+        }
+        expected_output = {
+            "safely_pipeline_termination": True,
+        }
+        get_completed_and_graded_mock.return_value = (
+            get_complete_and_graded_output["is_complete"],
+            get_complete_and_graded_output["is_graded"],
+        )
+
+        result = terminate_not_full_completion_cases(self.user_id, self.course_id, **pipeline_kwargs)
+
+        self.assertDictEqual(expected_output, result)
+
+    @patch("eox_nelp.pearson_vue.pipeline.get_completed_and_graded")
+    def test_check_complete_graded(self, get_completed_and_graded_mock):
+        """Test the pipeline return expected dict(safely_pipeline_termination) with empty `is_passing`, and
+           is_complete=True, is_graded=True.
+        Expected behavior:
+            - Returned value is dict with `safely_pipeline_termination`
+        """
+        pipeline_kwargs = {}
+        get_complete_and_graded_output = {
+            "is_complete": False,
+            "is_graded": True,
+        }
+        expected_output = {
+            "safely_pipeline_termination": True,
+        }
+        get_completed_and_graded_mock.return_value = (
+            get_complete_and_graded_output["is_complete"],
+            get_complete_and_graded_output["is_graded"],
+        )
+
+        result = terminate_not_full_completion_cases(self.user_id, self.course_id, **pipeline_kwargs)
+
+        self.assertDictEqual(expected_output, result)
```

Comment:
```suggestion
        self.assertIsNone(result)
```
That happens with  is_complete=True and is_graded=True

Reply:
I think here is dict with safely termination

Reply:
you'r right because it's `not is_graded`